src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -19,8 +19,8 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all candidates whose description contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + " The search will be conducted only on a specific field in candidates' description by specifying the"
-            + PREFIX_FIELD + " FIELD argument.\n"
+            + "The search will be conducted only on a specific field in candidates' description by specifying the "
```

Comment:
Thanks for helping to fix the whitespacing 😄

Reply:
No problem at all! Char wait! More stuff to add